lightning/src/ln/channel.rs

```diff
@@ -2174,8 +2189,13 @@ impl FundingScope {
 /// Info about a pending splice, used in the pre-splice channel
 #[cfg(splicing)]
 struct PendingSplice {
+	/// Intended contributions to the splice from our end
 	pub our_funding_contribution: i64,
 	funding: Option<FundingScope>,
+	funding_negotiation_context: FundingNegotiationContext,
```

Comment:
Do we keep this around after negotiation completes so that it can be used for RBF?

Reply:
In this PR not, as this PR covers the use case only until the negotiation completes. It should be done that way in the next steps.